internal/uvm/vpmem_mapped.go

```diff
@@ -234,10 +237,11 @@ func (uvm *UtilityVM) addVPMemMappedDevice(ctx context.Context, hostPath string)
 		return dev.uvmPath, nil
 	}
 
-	devSize, err := fileSystemSize(hostPath)
+	st, err := os.Stat(hostPath)
 	if err != nil {
 		return "", err
 	}
+	devSize := pageAlign(uint64(st.Size()))
```

Comment:
why this change?

Reply:
On the guest side, I create a linear target using device mapper based on `devSize`.
Since the dm-verity hash device will be appended after ext4 data, I need to know the size of the entire thing, otherwise the resulting linear target won't have dm-verity information.

Reply:
Could you add a comment saying that?